go/src/koding/kites/kloud/provider/marathon/machine.go

```diff
@@ -0,0 +1,52 @@
+package marathon
+
+import (
+	"koding/kites/kloud/machinestate"
+	"koding/kites/kloud/stack"
+	"koding/kites/kloud/stack/provider"
+
+	"golang.org/x/net/context"
+)
+
+// Machine represents a single container within a Marathon app.
+type Machine struct {
+	*provider.BaseMachine
+}
+
+var (
+	_ provider.Machine = (*Machine)(nil) // public API
+	_ stack.Machiner   = (*Machine)(nil) // internal API
+)
+
+func newMachine(bm *provider.BaseMachine) (provider.Machine, error) {
+	m := &Machine{BaseMachine: bm}
+
+	return m, nil
+}
+
+// Starts starts the app, that is all containers.
+//
+// It does not support starting a single container within an app,
+// so stopping whichever container from the app stops the whole app.
+func (m *Machine) Start(ctx context.Context) (interface{}, error) {
+	return nil, nil
+}
+
+// Stop stops the app, that is all containers.
+//
+// It does not support stopping a single container within an app,
+// so stopping whichever container from the app stops the whole app.
+func (m *Machine) Stop(ctx context.Context) (interface{}, error) {
+	return nil, nil
+}
+
+// Info gives state of the app.
+func (m *Machine) Info(context.Context) (machinestate.State, interface{}, error) {
+	return machinestate.Running, nil, nil
```

Comment:
your app definition and mines are different 😄  i meant as app is `kloud/provider/marathon/machine.go` the code itself